### DISCUSSION TITLE
I've forgot my old mentions filter

### DISCUSSION THREAD

**Discussion by Born2Raf** [Chatterino/chatterino2#4380] [Q&A] [answered]
Hello, first of all, apologies for my approximative english and for this question which might be sound dumb to some. I'm not very familiar with coding but I had a filter who was allowing me to get mention for specific channels. 

To describe as best as I can : See a tab with live chat and a split with /mentions+filters. My filter command was something like : channel.name == "StreamerName" || channel.name == "MyUsername". 

I can't remember if it was && or || but the thing is, it doesnt work anymore since I've updated my chatterino to 2.4.0 nightly version. I had this filter for a few channels by simply replacing channel.name "StreamerName".

It was still showing up in my global /mentions tho but wasn't a big deal tbh.
If someone can drop me the exact filter command I had it would be really appreciated ! (I've searched the all web and github but didn't find it, maybe i'm blind, if so, apologies!!)

Thanks in advance and sorry for this newbie issue !
Best regards
Rafael

**Comment by Felanbird** [comment, marked as answer]
You'd likely want a filter like this, for mentions only from specific channels

`(channel.name == "streamerName" && message.content contains "myUsername")`

In this example, I'm using `channel.name == "pajlada" && message.content contains "felanbird"` and the `💎pir` highlight only appears in the non filtered tab
![image](https://user-images.githubusercontent.com/41973452/219100406-16b8b493-7e4c-452c-9af9-bcd358faa01e.png)

If you'd like to expand this to multiple channels, you can utilize `||` which means OR - while `&&` means AND. 
The brackets I've outlined are required for this one:

`((channel.name == "streamerOne" || channel.name == "streamerTwo" || channel.name == "streamerThree") && message.content contains "myUsername")`

**Reply by Born2Raf** [reply]
Thanks you very much !!! Will try this out as soon as possible !

Regarding the second command with StreamerOne Two Three, it does regroup the mentions of those 3 channels into one single mentions filtered right ? Correct me if my understanding is poor ! 

Another quick question, does `channel.name == "Streamer" && message.content contains "myusername" || message.content contains "word1" || message.content contains "word2" ` would work aswell ? Or is it better to do it in separate filters ?

Have a very good one, thank you again!
Best regards
Rafael

(edit : screenshots attached shows it still ping me in mentions instead of filtered)
  _removed sc because fixed_

**Reply by Felanbird** [reply]
My immediate assumption would be you have another filtered applied to that tab

**Reply by Born2Raf** [reply]
Indeed ! Worked perfectly after I've removed the second filter ! Thank you again, for your time and patience !

**Reply by Felanbird** [reply]
As for your additional filter, that should work but I'd throw brackets on `(channel.name == "streamer")` just to be careful, usually less brackets breaks more often than extra brackets

**Reply by Born2Raf** [reply]
Lovely !! Works just fine ! Even better than my previous "setup" ! Less filters, much more efficient !! Thank you very much ! I'll be sure to save this before a future update so I won't have to bother you again !
